python/meterpreter/meterpreter.py

```diff
@@ -22,14 +22,6 @@
 else:
 	has_windll = hasattr(ctypes, 'windll')
 
-# this MUST be imported for urllib to work on OSX
```

Comment:
Thanks @zeroSteiner , I was thinking more in relation to this commit: https://github.com/rapid7/metasploit-payloads/commit/b5b1c9f83221979347e83582b6a8a7059ab63d03